tests/integration/targets/iosxr_config/tests/cli/commit_confirmed.yaml

```diff
@@ -5,6 +5,7 @@
   vars:
     ansible_iosxr_commit_confirmed: true
     ansible_iosxr_commit_confirmed_timeout: 90
+    ansible_iosxr_commit_label: testLabel1
```

Comment:
The test just checks the rollback behavior. Could you also add a test that verifies commit confirmed to be working?